### PR TITLE
feat(design-system,editor-ui): upgrade some of the frontend dev dependencies

### PR DESCRIPTION
**Pull request by netroy**

### CODE REVIEW

File: packages/design-system/postcss.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+	plugins: {
+    autoprefixer: {},
```
Comment (alexgrozav) [inline packages/design-system/postcss.config.js:3]: nitpick: Indentation issue here. Probably tabs vs. spaces issue.

File: packages/design-system/.storybook/main.js [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
 	stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
 	addons: [
 		'@storybook/addon-links',
 		'@storybook/addon-essentials',
+		{
+			name: '@storybook/addon-postcss',
+			options: {
+				postcssLoaderOptions: {
+          implementation: require('postcss'),
+        },
```
Comment (alexgrozav) [inline packages/design-system/.storybook/main.js:16]: nitpick: Indentation issue here. Probably tabs vs. spaces issue.

**Review by alexgrozav — DISMISSED** [review]
Awesome work! Thanks for taking care of this. 🙌

File: packages/design-system/postcss.config.js (hunk 1 above)
Comment (netroy) [reply packages/design-system/postcss.config.js]: fixed. can you please approve again :pray:

**Review by alexgrozav — APPROVED** [review]

**Pull request merged**